src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/Jira.kt

```diff
@@ -11,10 +12,10 @@ import org.apache.logging.log4j.Logger
 import java.net.URI
 import java.util.concurrent.Executors
 
-class Jira(
+class Jira private constructor(
     private val nodes: List<StartedNode>,
     val jiraHome: RemoteLocation,
-    val database: RemoteLocation?,
```

Comment:
Are there any plans for support of databases that don't have file-based backups?